clang/include/clang/Basic/AttrDocs.td

```diff
@@ -3155,6 +3155,16 @@ implicitly inherit this attribute.
   }];
 }
 
+def SYCLGlobalVarDocs : Documentation {
+  let Category = DocCatFunction;
+  let Heading = "SYCL Global Var";
```

Comment:
```suggestion
```
IIRC, there's not a need to specify a heading when the attribute has a simple spelling. You only need this when the attribute has multiple spellings and you want to pick something a bit different than a laundry list of syntaxes.